MainModule/Client/UI/Default/PerfStats.lua

```diff
@@ -41,7 +41,7 @@ return function(data,env)
 		window:Ready()
 
 		repeat
-			label.Text = `Render: {math.round(1/service["RunService"].RenderStepped:Wait())} fps\nPhysics: {math.round(workspace:GetRealPhysicsFPS())} fps`
+			label.Text = `Render: {math.round(1/service["RunService"].RenderStepped:Wait())} fps\nPhysics: {math.round(workspace:GetRealPhysicsFPS())} fps\nPing: {service.Players.LocalPlayer:GetNetworkPing()}ms`
```

Comment:
for fucksake math.round really who did that do math.floor instead

Reply:
What?

Reply:
Why? math.floor() would always return 59 even if the fps is 59.999999999999999

It should report the closest not the floor

Reply:
> you stupid? math.round isnt accurate enough thats why we math.floor

`math.floor` is less accurate and `math.round` is moree accurate

0.9999, floor = 0, round = 1
1.1111, floor = 1, round = 1
1.4444, floor = 1, round = 1
1.5555, floor = 1, round = 2
1.9999, floor = 1,  round = 2

Reply:
nuh

Reply:
if its 1.9999 then its still not 2 smh its 1.9999

Reply:
> if its 1.9999 then its still not 2 smh its 1.9999

Why should it always round it down instead of the closest? May you elaborate?

Reply:
bc its more accurate thats how the fps thingy works on Nvidia Alt + R thingymabob

Reply:
😐

Reply:
Let's do a compromise and display the first dedimal like Roblox does

Everyone is happy